src/components/game/AddHighscore.tsx

```diff
@@ -1,18 +1,20 @@
-import { Box, Button, Stack, Text } from '@chakra-ui/react';
-import { useNavigate } from 'react-router-dom';
-import { Input } from '@chakra-ui/react';
-
-import { Center } from '@chakra-ui/react';
-import { useState } from 'react';
-import { submitScore } from '../../services/game';
-import '../../fonts.css';
+import { Box, Button, Center, Input, Stack, Text } from '@chakra-ui/react';
 import { AnimatePresence, motion } from 'framer-motion';
+import { useEffect, useState } from 'react';
+import '../../fonts.css';
+import Highscores from '../../pages/Highscores';
+import { submitScore } from '../../services/game';
 import './Game.css';
 
+interface AddHighscorePropsI {
+    points: number;
+    playAgain: () => void;
+}
+
 /**
  * Function that takes the points as a prop and returns a component that handles logic for submitting the score
  */
-function AddHighscore(props: { points: number }) {
+function AddHighscore(props: AddHighscorePropsI) {
```

Comment:
`const AddHighscore: React.FC<AddHighscoreProps> = ({points, playAgain}) => {..}` for konsistent kodestil

Reply:
Done